cloudflared/config.yaml

```diff
@@ -16,12 +16,14 @@ arch:
   - armv7
   - i386
 options:
+  quick_tunnel: false
   external_hostname: ""
   tunnel_name: "homeassistant"
   additional_hosts: []
 schema:
-  external_hostname: str
-  tunnel_name: str
+  quick_tunnel: bool
```

Comment:
I think we should make the `quick_tunnel` variable optional here and not define any default value in the options. This way, we can offer the option (also prominently in the Docu) without making it required for the regular user.

Reply:
Agreed, have made the change